lib/src/okta/okta.module.ts

```diff
@@ -44,6 +44,9 @@ import { OktaConfig, OKTA_CONFIG, OKTA_AUTH } from './models/okta.config';
         [new Optional(), Location]
       ]
     },
+    // Provide empty OKTA_CONFIG by default
```

Comment:
I don't think this is necessary now that the `OktaCallbackComponent` injects the `OktaAuthConfigService`. 
I tested my sample project (mimicking code from the PR) via providing `OKTA_CONFIG`, using `.forRoot`, and with runtime config loading (`APP_INITIALIZER`).